lms/templates/header/header.html

```diff
@@ -42,79 +38,29 @@
 % if site_status_msg:
   <div class="site-status">
     <div class="inner-wrapper">
-      <span class="icon fa fa-warning"></span>
+      <span class="icon fa fa-warning" aria-hidden="true"></span>
       <p>${site_status_msg}</p>
     </div>
   </div>
 % endif
 </%block>
 
+<header class="global-header ${'slim' if course else ''}">
+  <%include file="navbar-logo-header.html" args="online_help_token=online_help_token"/>
+  <div class="hamburger-menu" role="button" aria-label="Options Menu" aria-expanded="false" aria-controls="mobile-menu" tabindex="0">
```

Comment:
String needs to be wrapped for translations.
Is there a reason that this is a ```div``` with ```role="button"``` instead of just a ```button``` element?

Reply:
It seemed like a simpler implementation than overriding all the button styling that gets tied to the element.

Reply:
I'm fine with shipping this as is for now to get this out, but I would prefer to utilize the correct HTML elements and update Sass than force a div to act like a button. Because this is a pattern we use throughout the site (buttons that look like link) there would be value in adding a class to easily style buttons as links (not necessarily in this PR).

Reply:
There is a class `.btn-link` that works in at least some of the styles. It wouldn't take too much to get it to work in all three.